Sources/MapboxSearch/PublicAPI/Use Cases/Address Autofill/AddressAutofill.swift

```diff
@@ -197,56 +200,59 @@ private extension AddressAutofill {
         with options: CoreRequestOptions,
         completion: @escaping (Swift.Result<[Suggestion], Error>) -> Void
     ) {
-        guard !suggestions.isEmpty else {
-            return completion(.success([]))
+        let resolvedSuggestions = suggestions.compactMap { result -> Suggestion? in
+            guard let address = result.addresses?.first,
+                  let resultAddress = try? address.toAutofillComponents() else {
+                return nil
+            }
+
+            let matchingName = result.matchingName ?? " "
```

Comment:
Is `matchingName` expected to be used as `name`? Is ` " "` valid default value?

Reply:
I'll change this to `guard let name = result.names.first` so we always know that a valid name is available for non-nil results.